src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.cs

```diff
@@ -418,8 +418,9 @@ private async Task PerformDisconnectAsyncCore(string connectionId)
                 connection.AbortOnClose = false;
 
                 // We're done writing to the application output
-                connection.Application.Output.CancelPendingFlush();
+                connection.Application.Output.Complete();
```

Comment:
Errr not sure

Reply:
This happens to work because as we are using Unbounded pipeline, `WriteAsync` to this `Connection.Application.Output` is synchronous...... So this line must be triggered after all the `WriteAsync` action....

Reply:
I don't like it either....